cmd/kubent/main.go

```diff
@@ -91,9 +106,18 @@ func main() {
 
 	log.Info().Msg(">>> Kube No Trouble `kubent` <<<")
 	log.Info().Msgf("version %s (git sha %s)", version, gitSha)
-	log.Info().Msg("Initializing collectors and retrieving data")
 
+	log.Info().Msg("Initializing collectors and retrieving data")
 	initCollectors := initCollectors(config)
+
+	version, err := getServerVersion(initCollectors)
+	if err != nil {
+		log.Fatal().Err(err).Msg("Failed to detect K8s version")
+	}
+	if version != nil {
```

Comment:
this should be `else if`, but probably will further change once I add the overriding flag logic.

Reply:
or or

Reply:
changed